test/loki_service.cc

```diff
@@ -14,13 +14,16 @@
 
```

Comment:
i added tests for both valhalla and osrm formats for the status endpoint and i copy pasted the tests from thor service into here for posterity. at somepoint we can test those too when we refactor this to be an integration service test